### PR TITLE
fix(agw): sctpd needs explicit restart for docker as it is restarted …

### PR DESCRIPTION
**Pull request by crasu**
## Summary

* Added explicit sctpd to clear redis as with systemd the restart is triggered implictly anyway
* Removed loop that is only run for one iteration

## Test Plan

* Ran the integ test

### CODE REVIEW

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py [hunk 1]
```diff
@@ -49,56 +49,49 @@ def test_attach_detach_with_corrupt_stateless_mme(self):
...
+        )
+
+        # Wait on EMM Information from MME
+        self._s1ap_wrapper._s1_util.receive_emm_info()
+
+        print("************************* Corrupting sctpd state")
+        self._s1ap_wrapper.magmad_util.corrupt_agw_state("sctpd")
+
+        print("************************* Restarting %s service sctpd")
+        self._s1ap_wrapper.magmad_util.restart_services(["sctpd"], wait_time=60)
```
Comment (mpfirrmann) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py:61]: `%s` is not needed here.

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py [hunk 2]
```diff
@@ -49,56 +49,49 @@ def test_attach_detach_with_corrupt_stateless_mme(self):
             "UE id ", req.ue_id,
         )
 
-        for s in services_state_dict:
-            # Now actually complete the attach
```
Comment (mpfirrmann) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py:53]: `services_state_dict` definition in lines 42-44 can be removed, as well.

**Review by mpfirrmann — CHANGES_REQUESTED** [review]

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py [hunk 3]
```diff
@@ -1061,7 +1061,7 @@ def restart_services(self, services, wait_time=0):
 
     def restart_mme(self, wait_time=20):
         """
-        Restart MME service and wait for the service to come up properly
+        Restart MME service
```
Comment (VinashakAnkitAman) [inline lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: We are still waiting for the services to come-up after restart inside this function in line number 1073. Is this change needed?

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py [hunk 4]
```diff
@@ -49,56 +49,49 @@ def test_attach_detach_with_corrupt_stateless_mme(self):
...
+        self._s1ap_wrapper._s1_util.attach(
+            req.ue_id, s1ap_types.tfwCmd.UE_END_TO_END_ATTACH_REQUEST,
+            s1ap_types.tfwCmd.UE_ATTACH_ACCEPT_IND,
+            s1ap_types.ueAttachAccept_t,
+        )
+
+        # Wait on EMM Information from MME
+        self._s1ap_wrapper._s1_util.receive_emm_info()
+
+        print("************************* Corrupting sctpd state")
```
Comment (VinashakAnkitAman) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py]: As per the testcase description we are supposed to corrupt the mme state.

**Review by VinashakAnkitAman — CHANGES_REQUESTED** [review]

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py (hunk 3 above)
Comment (crasu) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: This line is going to be removed by another pr. Lets keep the comment for this pr.

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py (hunk 2 above)
Comment (crasu) [reply lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py:53]: done

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py (hunk 1 above)
Comment (crasu) [reply lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py:61]: done

File: lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py (hunk 4 above)
Comment (crasu) [reply lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py]: fixed

**Review by mpfirrmann — APPROVED** [review]

**Review by VinashakAnkitAman — APPROVED** [review]
LGTM. Please rebase the PR. CI check is failing.
